Documentation/Architecture/SpatialAwareness/SpatialAwarenessSystemArchitecture.md

```diff
@@ -2,42 +2,68 @@
-- [MixedRealitySpatialAwarenessMeshHandler](MixedRealitySpatialAwarenessMeshHandler.md)
-- [MixedRealitySpatialAwarenessPlanarSurfaceHandler](MixedRealitySpatialAwarenessPlanarSurfaceHandler.md)
 
 ## Enumerations
 
-- [SpatialAwarenessMeshDisplayOptions](SpatialAwarenessMeshDisplayOptions.md)
-- [SpatialAwarenessMeshLevelOfDetail](SpatialAwarenessMeshLevelOfDetail.md)
-- [SpatialAwarenessSurfaceTypes](SpatialAwarenessSurfaceTypes.md)
+- [SpatialAwarenessMeshDisplayOptions](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.SpatialAwarenessMeshDisplayOptions)
+- [SpatialAwarenessMeshLevelOfDetail](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.SpatialAwarenessMeshLevelOfDetail)
+- [SpatialAwarenessSurfaceTypes](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.SpatialAwarenessSurfaceTypes)
 
-## Event Data Types
 
-- [MixedRealitySpatialAwarenessEventData](MixedRealitySpatialAwarenessEventData.md)
+## Events and Event Data
 
-## Configuration Profile
+- [IMixedRealitySpatialAwarenessObservationHandler<T>](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.IMixedRealitySpatialAwarenessObservationHandler-1)
+- [MixedRealitySpatialAwarenessEventData](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessEventData)
+- [MixedRealitySpatialAwarenessEventData<T>](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessEventData-1)
+
+## Profiles
+
+- [MixedRealitySpatialAwarenessSystemProfile](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessSystemProfile)
+- [BaseSpatialAwarenessObserverProfile](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.BaseSpatialAwarenessObserverProfile)
+- [MixedRealitySpatialAwarenessMeshObserverProfile](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessMeshObserverProfile)
 
-- [MixedRealitySpatialAwarenessProfile](MixedRealitySpatialAwarenessProfile.md)
-- [MixedRealitySpatialAwarenessProfileInspector](MixedRealitySpatialAwarenessProfileInspector.md)
- 
 ## Example Implementation
 
-The Mixed Reality Toolkit's default implementation of the spatial awareness system can be found in the Assets/MixedRealityToolkit/SDK/Features/SpatialAwarenessSystem folder. This implementation can be used as an example of how to build your own spatial awareness system. It is well documented and demonstrates each of the features described in this architecture specification.
+### Service
+
+The default spatial awareness system is a good example for developers looking to create their own implementation or to learn more about how the system works. It is provided as part of the Mixed Reality Toolkit Spatial Awareness service package. 
+
+The latest released code is available on [GitHub](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development_hl2/Assets/MixedRealityToolkit.Services/SpatialAwarenessSystem/MixedRealitySpatialAwarenessSystem.cs).
+
```

Comment:
Unless the plan is to flesh these out in the next change, this section (and the next class documentation section) don't really add that much to this page - it leaves me wondering more than it tells me.

Reply:
do you recommend removing the example implementation section?

Reply:
xref references (suggested by @thalbern ) are resolved by docfx, I believe